core/converters.go

```diff
@@ -96,3 +97,27 @@ func SecondsToHourMinSec(input int) string {
 
 	return result
 }
+
+// EpochStartIdentifier returns the string for the epoch start identifier
+func EpochStartIdentifier(epoch uint32) string {
+	return fmt.Sprintf("epochStartBlock_%d", epoch)
```

Comment:
maybe set this in a constant in core.constants and use it also in the tests if there are any

Reply:
everywhere epochStartIdentifier is used, is through this method. there is not recreation in other places.

Reply:
ok